packages/orbit-components/src/Pagination/Pagination.stories.tsx

```diff
@@ -12,16 +12,33 @@ export default {
 
 export const Default = () => {
   const pageCount = number("pageCount", 6);
+  const labelPrev = text("labelPrev", "prev");
+  const labelNext = text("labelNext", "next");
+  const labelProgress = text("labelProgress", "of");
 
-  return <Pagination pageCount={pageCount} onPageChange={action("onPageChange")} />;
+  return (
+    <Pagination
+      labelPrev={labelPrev}
+      labelNext={labelNext}
+      labelProgress={labelProgress}
+      pageCount={pageCount}
+      onPageChange={action("onPageChange")}
+    />
+  );
 };
 
 export const WithALotOfPages = () => {
   const selectedPage = number("selectedPage", 44);
   const pageCount = number("pageCount", 100);
+  const labelPrev = text("labelPrev", "prev");
+  const labelNext = text("labelNext", "next");
+  const labelProgress = text("labelProgress", "of");
 
   return (
     <Pagination
+      labelPrev={labelPrev}
+      labelNext={labelNext}
+      labelProgress={labelProgress}
```

Comment:
This should be something like
```suggestion
      labelProgress={`${selectedPage} ${labelProgress} ${pageCount}`}
```
And the same for all other stories